.github/workflows/lint.yml

```diff
@@ -27,7 +27,7 @@ jobs:
             **/**.go
             go.mod
             go.sum
-      - uses: golangci/golangci-lint-action@v3.4.0
+      - uses: golangci/golangci-lint-action@v3.5.0
         with:
           version: v1.50.1
```

Comment:
I'm not sure, but, don't we need to use `1.53.1`?
* https://github.com/Finschia/ostracon/pull/626

Reply:
https://github.com/cometbft/cometbft/blob/028fb5eb0c071ec2c5fea34fe3d02b1fea814d6d/.github/workflows/lint.yml#L34
Its field is specified "latest" in cometBFT,  so we do so.

Reply:
Please try once with `latest`. But if we use `latest` and fail the test, please ignore my comment.

Reply:
I've also confirmed that v0.34.x is using `1.50.1`
* https://github.com/tendermint/tendermint/blob/v0.34.24/.github/workflows/lint.yml
* https://github.com/cometbft/cometbft/blob/v0.34.28/.github/workflows/lint.yml

Reply:
Tests seem to works normally.